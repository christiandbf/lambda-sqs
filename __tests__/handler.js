const handler = require('../handler');

describe('handler.js - hello', () => {
  it('Should return test message', async () => {
    const response = await handler.hello({});
    expect(response).toMatchSnapshot();
  });
});
