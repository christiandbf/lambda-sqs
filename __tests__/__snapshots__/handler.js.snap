// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handler.js - hello Should return test message 1`] = `
Object {
  "body": "{
  \\"message\\": \\"Go Serverless v1.0! Your function executed successfully!\\",
  \\"input\\": {}
}",
  "statusCode": 200,
}
`;
